test/Project.toml

```diff
@@ -0,0 +1,24 @@
+[deps]
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+RealDot = "c1ae055f-0cd5-4b69-90a6-9a35b1a98df9"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+# and test-only:
+ChainRulesTestUtils = "cdddcdb0-9152-4a09-a978-84456f9df70a"
+FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000"
+JuliaInterpreter = "aa1ae85d-cabe-5617-a682-6adf51b2e16a"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[compat]
+ChainRulesCore = "1.11.5"
```

Comment:
Does test/Project.toml need to contain full-time deps again, or are they implied?

Reply:
They are implied but I would rather not use the `test/Project.toml`.
It is annoying to keep these things in sync when things move from test only to main dep.

Reply:
I don't mind, should have left it a separate commit... you can fight it out with Seth?

Reply:
Seth, this might be of interest: https://github.com/JuliaTesting/TestEnv.jl